### PR TITLE
Feature/shubham/login design

### PR DESCRIPTION
**Pull request by panic-coder**

### CODE REVIEW

File: admin/src/app/containers/default-layout/default-layout.component.html [hunk 1]
```diff
@@ -2,14 +2,16 @@
 <app-header
   [navbarBrandRouterLink]="['/orders/stock-orders']"
   [fixed]="true"
-  [navbarBrandFull]="{src: 'assets/img/logo.png', width: 89, height: 25, alt: 'ShoppinPal Logo'}"
-  [navbarBrandMinimized]="{src: 'assets/img/logo.png', width: 30, height: 30, alt: 'ShoppinPal Logo'}"
+  [navbarBrandFull]="{src: 'assets/img/logo.png', width: 100, height: 25, alt: 'ShoppinPal Logo'}"
```
Comment (kamal0808) [inline admin/src/app/containers/default-layout/default-layout.component.html]: change this logo to Stockup's logo

File: admin/src/app/containers/default-layout/default-layout.component.html [hunk 2]
```diff
@@ -2,14 +2,16 @@
...
+  [asideMenuToggler]="false"
   [sidebarToggler]="'lg'"
-  [asideMenuToggler]="'lg'">
+  [asideMenuToggler]="'lg'"
+  >
   <ul class="nav navbar-nav ml-auto">
     <li class="nav-item dropdown" dropdown placement="bottom right">
       <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true"
-         aria-expanded="false" dropdownToggle (click)="false">
+         aria-expanded="false" dropdownToggle (click)="false" style="padding-right: 10px;">
```
Comment (kamal0808) [inline admin/src/app/containers/default-layout/default-layout.component.html]: check if there's a margin/padding class

File: admin/src/app/home/file-imports/file-imports-routing.module.ts [hunk 3]
```diff
@@ -4,20 +4,37 @@ import {UserResolverService} from "../../shared/services/user-resolver.service";
...
 const routes: Routes = [
   {
-    path: 'file-imports',
+    path: '',
     resolve: {
       user: UserResolverService,
       access: AccessService
     },
     children: [
+      // {
```
Comment (kamal0808) [inline admin/src/app/home/file-imports/file-imports-routing.module.ts]: remove comments

File: admin/src/app/login/login.component.scss [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+  position: relative;
+  display: table; 
+
+  // position: relative;
+
+  // transform: translate(-50%, -50%);
+
+}
+
+.image-header p {
```
Comment (kamal0808) [inline admin/src/app/login/login.component.scss]: write css according to sass syntax https://sass-lang.com/documentation/syntax/parsing

File: admin/src/app/login/login.component.ts [hunk 5]
```diff
@@ -57,4 +105,14 @@ export class LoginComponent implements OnInit {
       });
   }
 
+  enterEvent(event) {
```
Comment (kamal0808) [inline admin/src/app/login/login.component.ts]: remove this

File: admin/src/app/shared/constants/utils.ts [hunk 6]
```diff
@@ -13,5 +13,15 @@ export default class Utils {
       return re.test(String(email).toLowerCase());
     }
   }
-}
 
+  static singleValidateEmail(email) {
```
Comment (kamal0808) [inline admin/src/app/shared/constants/utils.ts]: make changes in the original function.

File: admin/src/app/shared/constants/utils.ts [hunk 7]
```diff
@@ -13,5 +13,15 @@ export default class Utils {
...
+    var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return re.test(String(email).toLowerCase());
+  }
+
+  static validatePassword(password) {
+    var re = /^.{6,}$/;
```
Comment (kamal0808) [inline admin/src/app/shared/constants/utils.ts]: assign max password length to a variable

File: admin/src/app/containers/default-layout/default-layout.component.html (hunk 1 above)
Comment (panic-coder) [reply admin/src/app/containers/default-layout/default-layout.component.html]: done

File: admin/src/app/containers/default-layout/default-layout.component.html (hunk 2 above)
Comment (panic-coder) [reply admin/src/app/containers/default-layout/default-layout.component.html]: no such class found

File: admin/src/app/home/file-imports/file-imports-routing.module.ts (hunk 3 above)
Comment (panic-coder) [reply admin/src/app/home/file-imports/file-imports-routing.module.ts]: done

File: admin/src/app/login/login.component.scss (hunk 4 above)
Comment (panic-coder) [reply admin/src/app/login/login.component.scss]: done

File: admin/src/app/login/login.component.ts (hunk 5 above)
Comment (panic-coder) [reply admin/src/app/login/login.component.ts]: done

File: admin/src/app/shared/constants/utils.ts (hunk 7 above)
Comment (panic-coder) [reply admin/src/app/shared/constants/utils.ts]: done

File: admin/src/app/shared/constants/utils.ts (hunk 6 above)
Comment (panic-coder) [reply admin/src/app/shared/constants/utils.ts]: done

**Pull request merged**